tests/frontend/test_utils.py

```diff
@@ -580,7 +580,6 @@ def dummy_func(r, theta, phi, q):
 class TestEngineUtilityFunctions:
     """Tests for some engine auxiliary functions"""
 
-    @pytest.mark.skip('FIXME is_unitary')
```

Comment:
Perhaps I missed it, but I didn't see where else in the PR that changes were made which were connected to these failing tests 🤔

Reply:
I think I fixed the function back when I updated utils.py, but I forgot to enable the tests.